src/shared/utils/utils.tsx

```diff
@@ -33,8 +33,16 @@ export function removeItemFromList<T>(list: T[], index: number) {
   return [...list.slice(0, index), ...list.slice(index + 1)];
 }
 
-export const hasContent = (obj: Record<string | number | symbol, unknown>) =>
-  Object.values(obj).some((val) => {
+export const hasContent = (
+  obj: Record<string | number | symbol, unknown>,
+  exclude?: string
+) => {
+  // If certain property has to be excluded while checking for content (like section specific)
+  const objCopy = { ...obj };
+  if (exclude && objCopy[exclude]) {
+    delete objCopy[exclude];
+  }
+  return Object.values(objCopy).some((val) => {
```

Comment:
```suggestion
  return Object.entries(obj).some(([key, val]) => {
    // If certain property has to be excluded while checking for content (like section specific)
    if (key === exclude) {
      return false;
    }
```
Maybe, reading again the code, the intent might be clearer filtering this directly in the callback for `.some()`